src/pharmpy/tools/amd/run.py

```diff
@@ -322,12 +366,29 @@ def _run_modelsearch(model):
     return _run_modelsearch
 
 
+def _subfunc_structsearch(route, modeltype, emax_init, ec50_init, path) -> SubFunc:
+    def _run_structsearch(model):
+        res = run_tool(
+            'structsearch',
+            route=route,
+            type=modeltype,
+            emax_init=emax_init,
+            ec50_init=ec50_init,
+            model=model,
+            path=path / 'structsearch',
+        )
+        assert isinstance(res, Results)
+        return res
+
+    return _run_structsearch
+
+
 def _subfunc_iiv(path) -> SubFunc:
     def _run_iiv(model):
         res = run_tool(
             'iivsearch',
             'brute_force',
-            iiv_strategy='fullblock',
+            iiv_strategy='no_add',
```

Comment:
Won't this change the behaviour of regular pk models?

Reply:
I changed this since it says in the documentation that "no_add" is the default.

Reply:
Yes, that is why it has to be set to `fullblock` since this is what we want for pk models. I don't think it should be different for pd models.

Reply:
I will change it back to `fullblock`